CONTRIBUTING.md

```diff
@@ -38,10 +37,7 @@ give us permission to use your contributions.
 
        More detailed explanatory text, if necessary. Wrap it to 72 characters.
        Think of this like an email, where you have a subject line and a body.
-       Make sure you mention the bug being fixed on a line all by itself at the
-       end, like so:
 
-       LP: #<bug number>
```

Comment:
but I don't think of it as mandatory either anymore as I don't want to not trust people about not including them and triple checking